pkg/vip/address.go

```diff
@@ -466,6 +466,8 @@ func (configurator *network) removeIptablesRulesForMasquerade() error {
 	return nil
 }
 
+// TODO: investigate if adding "--vport <port>" would be better or not quite necessary
+// After this rule is added, ipvs kernel module is also loaded
 func addMasqueradeRuleForVIP(ipt *iptables.IPTables, vip, comment string) error {
 	err := ipt.InsertUnique(iptables.TableNat, iptables.ChainPOSTROUTING,
 		1, "-m", "ipvs", "--vaddr", vip, "-j", "MASQUERADE", "-m", "comment", "--comment", comment)
```

Comment:
This rule is quite interesting. After this iptables added, ip_vs kernel module will be loaded to current container host automatically like:

```
 sudo lsmod | grep ip_vs
ip_vs                 176128  1 xt_ipvs
nf_conntrack          172032  10 xt_conntrack,nf_nat,nfnetlink_cttimeout,xt_nat,openvswitch,nf_conntrack_netlink,xt_CT,nf_conncount,xt_MASQUERADE,ip_vs
nf_defrag_ipv6         24576  3 nf_conntrack,openvswitch,ip_vs
libcrc32c              16384  7 nf_conntrack,nf_nat,openvswitch,btrfs,nf_tables,raid456,ip_vs
```

Because the AddIP() happens before NewIPVSLB(),  kube-vip doesn't require users to load ipvs manually in advance anymore in masquerade mode.